db/migrate/20220913150720_create_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
```

Comment:
- [ ] Great job.a user should control the post with a postcounter. you are missing the post counter 
eg `t.integer :posts_counter`